lib/Notes/NoteForm/NoteView.js

```diff
@@ -239,8 +242,9 @@ export default class NoteView extends Component {
                   <Col xs={12}>
                     <KeyValue
                       label={<FormattedMessage id="stripes-smart-components.details" />}
-                      value={get(noteData, 'content', '-')}
-                    />
+                    >
+                      <div dangerouslySetInnerHTML={noteContentMarkup} />
```

Comment:
Do we have a way of ensuring that the markup is safe? White-listed tags etc to ensure there aren't `<script>` tags or similar?

Reply:
Hi @doytch, this markup is saved with the '<Editpr/>' component and it sanitize it removing the <script> tags.
And on this page we receive this markup from our backend and not user input nor the 3rd party. We rely on our backend.